ui/pages/groups/index.js

```diff
@@ -59,7 +60,7 @@ function AddGroupsDialog({ setOpen }) {
                   }
                 }}
                 value={name}
-                onChange={e => setName(e.target.value)}
+                onChange={e => setName(e.target.value.replace(/^\s/g, ''))}
```

Comment:
On trim leading spaces since we want users to enter spaces into the input field. Trailing spaces will be trimmed on submit

Reply:
We should put this comment in the code too. Regex isn't intuitive imo